core/src/main/java/com/scalar/db/common/CheckedDistributedStorageAdmin.java

```diff
@@ -0,0 +1,277 @@
+    }
+
+    try {
+      admin.dropTable(namespace, table);
+    } catch (ExecutionException e) {
+      throw new ExecutionException(
+          "Dropping the table failed: " + ScalarDbUtils.getFullTableName(namespace, table), e);
+    }
+  }
+
+  @Override
+  public void dropNamespace(String namespace) throws ExecutionException {
+    if (checkNamespace && !namespaceExists(namespace)) {
+      throw new IllegalArgumentException("Namespace does not exist: " + namespace);
+    }
+    if (!getNamespaceTableNames(namespace).isEmpty()) {
+      throw new IllegalArgumentException(
+          "Namespace is not empty: " + namespace + ", " + getNamespaceTableNames(namespace));
+    }
+
+    try {
+      admin.dropNamespace(namespace);
+    } catch (ExecutionException e) {
+      throw new ExecutionException("Dropping the namespace failed: " + namespace, e);
+    }
+  }
+
+  @Override
+  public void truncateTable(String namespace, String table) throws ExecutionException {
+    if (!tableExists(namespace, table)) {
+      throw new IllegalArgumentException("Table does not exist: " + namespace + "." + table);
+    }
+
+    try {
+      admin.truncateTable(namespace, table);
+    } catch (ExecutionException e) {
+      throw new ExecutionException(
+          "Truncating the table failed: " + ScalarDbUtils.getFullTableName(namespace, table), e);
+    }
+  }
+
+  @Override
+  public void createIndex(
+      String namespace, String table, String columnName, Map<String, String> options)
+      throws ExecutionException {
+    TableMetadata tableMetadata = getTableMetadata(namespace, table);
+    if (tableMetadata == null) {
+      throw new IllegalArgumentException(
+          "Table does not exist: " + ScalarDbUtils.getFullTableName(namespace, table));
+    }
+    if (!tableMetadata.getColumnNames().contains(columnName)) {
+      throw new IllegalArgumentException(
+          "Column does not exist: "
+              + ScalarDbUtils.getFullTableName(namespace, table)
+              + ", "
+              + columnName);
+    }
+
+    if (indexExists(namespace, table, columnName)) {
+      throw new IllegalArgumentException(
+          "Index already exists: "
+              + ScalarDbUtils.getFullTableName(namespace, table)
+              + ", "
+              + columnName);
+    }
+
+    try {
+      admin.createIndex(namespace, table, columnName, options);
+    } catch (ExecutionException e) {
+      throw new ExecutionException(
+          "Creating the index failed: "
+              + ScalarDbUtils.getFullTableName(namespace, table)
+              + ", "
+              + columnName,
+          e);
+    }
+  }
+
+  @Override
+  public void dropIndex(String namespace, String table, String columnName)
+      throws ExecutionException {
+    if (!tableExists(namespace, table)) {
+      throw new IllegalArgumentException("Table does not exist: " + namespace + "." + table);
```

Comment:
```suggestion
      throw new IllegalArgumentException("Table does not exist: " + ScalarDbUtils.getFullTableName(namespace, table));
```
Likewise.

Reply:
Fixed in fcb233812ead7d8c6a68a422a7467f70ba10c733. Thanks.